src/main/java/org/kuali/kfs/ksb/messaging/threadpool/KSBScheduledPool.java

```diff
@@ -16,25 +16,12 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.kuali.kfs.kim.impl.permission;
+package org.kuali.kfs.ksb.messaging.threadpool;
 
-import org.kuali.kfs.kim.impl.common.template.Template;
+import org.kuali.kfs.core.api.lifecycle.Lifecycle;
 
-/* Cornell Customization: backport redis*/
-public class PermissionTemplate extends Template {
+import java.util.concurrent.ScheduledExecutorService;
 
-    private static final long serialVersionUID = 1L;
-    
-    public static final String CACHE_NAME = "PermissionTemplate";
+public interface KSBScheduledPool extends ScheduledExecutorService, Lifecycle {
```

Comment:
I am not sure how to fix this git issue, do you have some ideas? Thank you

Reply:
I'm not sure exactly how to fix this, unless there's some way to do so through an interactive rebase. If it ends up being more trouble than it's worth, I'm okay with leaving this Git setup as-is since the appropriate file changes/deletions appear to be in place.